.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
         commonjs: true,
         es6: true,
         node: true,
+        jest: true,
```

Comment:
it'll be needed wherever we have tests – i guess this is sort of prefiguring a world where all our tests sit alongside the module they test i.e. not just in the app?